README.md

```
@@ -60,17 +64,25 @@
 
 ## To simply try out the app
 1. Install [docker](https://docs.docker.com/get-docker/) and [docker compose](https://docs.docker.com/compose/install/)
-2. Start Django application and MongoDB using docker
+
+
+2. Create a .env file in your project's root directory by copying the contents from .env.example.
+
+3. Verify or update MONGO_REPLICA_HOST in your newly created .env file to ensure it is set as follows:
+    ``` env
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)